Description: Upstream changes introduced in version 4.4.0.20111005-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 serna-free (4.4.0.20111005-1) UNRELEASED; urgency=low
 .
   * Start working on serna 4.4
 .
 The person named in the Author field signed this changelog entry.
Author: Mathieu Malaterre <mathieu.malaterre@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: serna-free-4-4/serna/app/Makefile.pro
===================================================================
--- serna-free-4-4.orig/serna/app/Makefile.pro	2011-10-29 17:21:11.000000000 +0200
+++ serna-free-4-4/serna/app/Makefile.pro	2011-10-29 17:27:05.000000000 +0200
@@ -125,6 +125,8 @@
 
 unix:LIBS    += $$RTLIBS
 
+unix:LIBS    += -lqscintilla2
+
 SOURCES_PATTERN *= .cxx$
 win32:SOURCES_PATTERN *= win32/\.cxx$
 
Index: serna-free-4-4/serna/app/MANIFEST.speller
===================================================================
--- serna-free-4-4.orig/serna/app/MANIFEST.speller	2011-10-29 17:21:11.000000000 +0200
+++ serna-free-4-4/serna/app/MANIFEST.speller	2011-10-29 17:27:27.000000000 +0200
@@ -1,8 +1,8 @@
 #
 # Files which are required for speller plugin
 #
-include:${THIRD_DIR}/aspell/MANIFEST
-include:${THIRD_DIR}/hunspell/MANIFEST
+#include:${THIRD_DIR}/aspell/MANIFEST
+#include:${THIRD_DIR}/hunspell/MANIFEST
 
 [all]
 
Index: serna-free-4-4/serna/dll/Makefile.pro
===================================================================
--- serna-free-4-4.orig/serna/dll/Makefile.pro	2011-10-29 17:21:15.000000000 +0200
+++ serna-free-4-4/serna/dll/Makefile.pro	2011-10-29 17:27:05.000000000 +0200
@@ -42,7 +42,7 @@
                   msvcrt$$LIBDBG.lib \
                   advapi32.lib gdi32.lib shell32.lib ole32.lib oleaut32.lib
 
-linux:LIBS      = -lqt-mt -lsp -lantlr -lqassistantclient
+linux:LIBS      = -lqt-mt -lsp -lantlr -lqassistantclient -lqscintilla2
 linux:SYS_LIBS  = -ldl -lutil
 
 win32:LINK_FLAGS = -nologo -out:$@ -dll $$W32_LINK_FLAGS
Index: serna-free-4-4/serna/core/Makefile.pro
===================================================================
--- serna-free-4-4.orig/serna/core/Makefile.pro	2011-10-29 17:21:11.000000000 +0200
+++ serna-free-4-4/serna/core/Makefile.pro	2011-10-29 17:27:05.000000000 +0200
@@ -68,6 +68,6 @@
 ui_PATTERN      = $$UI_DIRS/.*\.ui$
 ALL_DEPS += $(msg_h) core_resource_data.cpp
 
-USE            = QtCore QtGui QtNetwork Qt3Support qtsingleapplication
+USE            = QtCore QtGui QtNetwork Qt3Support qtsingleapplication qscintilla
 
 EXTRA_TEMPLATES = genlist resource2cxx
Index: serna-free-4-4/serna/plugins/speller/Makefile.pro
===================================================================
--- serna-free-4-4.orig/serna/plugins/speller/Makefile.pro	2011-10-29 17:21:16.000000000 +0200
+++ serna-free-4-4/serna/plugins/speller/Makefile.pro	2011-10-29 17:27:05.000000000 +0200
@@ -37,6 +37,7 @@
 USE = QtGui QtCore QtXml QtNetwork 
 
 syspkg:USE += aspell
+unix:LIBS    += -laspell
 
 HEADERS_PATTERN = \.h$ \
                   aspell/.*\.h$ \
Index: serna-free-4-4/serna/plaineditor/Makefile.pro
===================================================================
--- serna-free-4-4.orig/serna/plaineditor/Makefile.pro	2011-10-29 17:21:16.000000000 +0200
+++ serna-free-4-4/serna/plaineditor/Makefile.pro	2011-10-29 17:27:05.000000000 +0200
@@ -26,7 +26,7 @@
         $(top_builddir)/serna/lib/docview \
         $(top_builddir)/serna/lib/docutils 
 
-USE             = QtCore QtGui Qt3Support QtXml QtNetwork qscintilla
+USE             = QtCore QtGui Qt3Support QtXml QtNetwork qscintilla qscintilla2
 
 HEADERS_PATTERN =   \.h$ \
                     impl/.*\.h$ \
Index: serna-free-4-4/serna/app/MANIFEST.docbook
===================================================================
--- serna-free-4-4.orig/serna/app/MANIFEST.docbook	2011-10-29 17:27:42.000000000 +0200
+++ serna-free-4-4/serna/app/MANIFEST.docbook	2011-10-29 17:27:46.000000000 +0200
@@ -349,4 +349,4 @@
     xh.xml
     lt.xml
 
-include:${THIRD_DIR}/docbook/MANIFEST
+#include:${THIRD_DIR}/docbook/MANIFEST
